src/Mod/Cloud/Gui/Command.cpp

```diff
@@ -0,0 +1,63 @@
+/***************************************************************************
+ *   Copyright (c) 2019         jmverdun3@gmail.com	                   *

```

Comment:
please add you name before the email and fix whitespace after it

Reply:
shall be fixed